src/data/local/github/tracks-node/track-6/steps/step-2.md

```diff
@@ -0,0 +1,32 @@
+## Step 2 - Pagination
+#### Estimated time: 4 hours
+
+Now, let's increment your `Users` query with a nice feature: pagination. Let's say the clients using your server want to fetch a list of users and there are lots of them on database. It's not a good practice to return too much users on a single request, because generally the client is not interested in using all of them. So, the solution is to offer separated small parts of users, according to some conditions the client sends.
```

Comment:
```suggestion
Now, let's increment your `Users` query with a nice feature: pagination. Let's say the clients using your server want to fetch a list of users and there are lots of them on database. It's not a good practice to return too many users on a single request, because generally the client is not interested in using all of them. So, the solution is to offer separated small parts of users, according to some conditions the client sends.
```

Acredito que outra motivação é não sobrecarregar o banco de dados e utilizar muito a rede